modules/javafx.graphics/src/main/java/javafx/scene/layout/BackgroundPosition.java

```diff
@@ -52,7 +52,7 @@
  * the Region's right edge.
  * @since JavaFX 8.0
  */
-public class BackgroundPosition implements Interpolatable<BackgroundPosition> {
+public final class BackgroundPosition implements Interpolatable<BackgroundPosition> {
```

Comment:
should we also declare the fields `final` as well?
horizontalSide, etc.

Reply:
All fields of `BackgroundPosition` are already final. Same for `BorderImage` and `BorderStroke`.

Reply:
sorry, I meant `private`

Reply:
Yes, that's a sensible change.